modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala

```diff
@@ -89,7 +89,7 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, q: engine.EventQueue, se:
             val user = userInRequest(req)
             WS(
               Exchange(
-                Process.emit(Binary(trimmedArray(SeqexecConnectionOpenEvent(user)))) ++
+                Process.emit(Binary(newTrimmedArray(ConnectionOpenEvent(user)))) ++
```

Comment:
This is the key change that will send proper authentication to the client upon connection